pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -193,6 +199,19 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 	// working with a copy to avoid race condition between this and resource version controller
 	toBeProcessedSVM := svm.DeepCopy()
 
+	gvr := getGVRFromResource(toBeProcessedSVM)
+
+	isUpdatable, err := svmc.isResourceUpdatable(gvr)
+	if err != nil {
+		return err
+	}
+
+	if !isUpdatable {
+		err := fmt.Errorf("resource %q does not support update verb", gvr.String())
+		logger.Error(err, "resource is not able to be migrated, not retrying", "gvr", gvr.String())
```

Comment:
You need to mark the migration as failed since this is terminal.

Reply:
Done!